README.md

```diff
@@ -141,6 +141,20 @@ Commands:
   run
 ```
 
+## Docker Usage
+
+Using an online cluster:
```

Comment:
What is an `online` cluster? 🤔

Reply:
do you want it rephrased to `on the internet`?

Reply:
I don't get the point why the cluster must be "on the internet" ?

Reply:
Any reason this was resolved ?